### DISCUSSION TITLE
Shared Scanners

### DISCUSSION THREAD

**Discussion by hoffadg** [chuot/rdio-scanner#317] [Q&A]
Is it possible to add systems from a shared scanner (ex. from broadcastify.com)?  I found some good air traffic control feeds near where I live and it would be great if I could somehow plug those into rdio-scanner and have them show up as a separate system.

Thanks,
Dan

**Comment by TPSSCW** [comment]
I'm doing that now with zello by using a old phone to listen to the feed and use the headphone jack to send it to a usb sound card. Trying to figure out how to do the same on rdio-scanner

**Comment by chuot** [comment]
To my knowledge, there is no tool to extract audio from broadcastify to send to Rdio Scanner

**Reply by hoffadg** [reply]
See my post below.  I created a "hacky" solution that seems to do the job :)

**Comment by TPSSCW** [comment]
I actually think it will work the same way as I'm doing it now by using a 
android tablet to play the broadcastify audio and pipe that into a usb 
sound card using a headphone Jack then capture in that USB sound card audio 
and pushing it out rdio scanner via Vox call. Do you know if voxcall can 
run multiple api key instances at the same time?

**Reply by hoffadg** [reply]
Ideally there would be a tool with the capability to receive one or more streams and pipe them to .mp3s with the proper naming depending on the source.  I figure someone has done something like this already..

**Reply by AvalancheAddict** [reply]
@TPSSCW  - You can definitely run multiple instances of VOXCALL at the same time with different key instances if you run them from different directories.  I have 4 copies of VOXCALL running simultaneously, pulling audio from 4 different sound cards, and feeding to 4 different talkgroups in RdioScanner.  You could probably do what you are doing right from the PC (without the tablet) by using a virtual audio cable.

**Comment by hoffadg** [comment]
Ok so I got tired of looking for something that does this so I came up with a windows batch script and python script that connect to one or many LiveATC.net streams, creates mp3 files every 2 minutes, using ffmpeg processes the files into chunks using dead space detection and then names each file properly so they can be ingested into rdio-scanner.  It's a total hack job but if anyone is interested, here it is.  You will need to satisfy all the dependencies and modify the hard coded file paths to make it work for your system.

I've been using it for a couple of hours and it works great!  Hearing both sides of the conversation on all channels just like I have an antenna set up at the end of the runway!

[stream_to_rdio-scanner.zip](https://github.com/chuot/rdio-scanner/files/10514246/stream_to_rdio-scanner.zip)

**Comment by renfrewcountyscanner** [comment]
You could do it with this.  https://github.com/aaknitt/voxcall
It can grab from any sound card and then upload it to Rdio.
